README.md

```diff
@@ -0,0 +1,40 @@
+
+| Title | KOBSearch |
+|--- | ---|
+| Version | TBD |
+
+
+### Abstract   
+
+The KOBSearch is a search service that provides a registered user with the information about the organizations (or members of the organization) that are operating in the city of Cochin and registered with KochiOrgBook (KOB). KOB is like a directory that will contain the verifiable data about the organizations thus enabling trusted digital verification service.
```

Comment:
- "(or members of the organization)" - You can take it out of the bracket.
- KOB is not the directory. It will be KOBAW( KOB Association Wallet) which is a wallet.
- The search can be done at 2 levels - 1) at wallet level, which will be details about the organizations and 2) at VCR level, which will contain details about entities inside the organization.